### PR TITLE
Increase input field width for varchar

### LINKED ISSUE #13627: Increase input field width for varchar(255)
The input field width (on Insert Row page) is currently 40 chars, could this be increased?
Perhaps the value fields could use all remaining width in the table.

### PR DESCRIPTION
**Pull request by aroralakshya**
Fixes #13627

Signed-Off-By: Lakshay arora <arora.lakshya123@gmail.com>

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
@nulll-pointer Thanks for your contribution!
You need to fix the tests before we can merge your pull request.
See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/390383283#L697

**Comment by mauriciofauth** [comment]
I have the impression that [`$cfg['CharTextareaCols']`](https://docs.phpmyadmin.net/en/latest/config.html#cfg_CharTextareaCols) no longer works with this patch. Is this expected?

**Comment by nulll-pointer** [comment]
"I have the impression that $cfg['CharTextareaCols'] no longer works with this patch. Is this expected?" 
Yes, this shouldn't work, I've set the width of the textarea to 90% of the page width(no matter what the configuration settings).
Should I set a minimum to the number of cols instead??

**Comment by nulll-pointer** [comment]
I have made some changes in the tests for insertEdit, kindly have a look.

**Comment by nulll-pointer** [comment]
I checked, this configuration isn't used anywhere else, I think using css to override it would be an overkill. Changing the default value sounds better.

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

File: libraries/classes/InsertEdit.php [hunk 1]
```diff
@@ -980,8 +980,8 @@ private function getTextarea(
              * @todo clarify the meaning of the "textfield" class and explain
              *       why character columns have the "char" class instead
              */
-            $the_class = 'char';
-            $textAreaRows = $GLOBALS['cfg']['CharTextareaRows'];
+            $the_class = 'char charField';
+            $textAreaRows = max($GLOBALS['cfg']['CharTextareaRows'], 7);
```
Comment (ibennetch) [inline libraries/classes/InsertEdit.php]: Is max() really a good choice here? What if (it's extremely unlikely, but still) a user deliberately wants more than 7 rows?

**Pull request merged**